appveyor.yml

```diff
@@ -10,14 +10,31 @@ init:
 configuration:
 - Debug
 - Release
+environment:
+  matrix:
+    - image_flag: 2015
+    - image_flag: 2017
+matrix:
+  exclude:
+    - image: Visual Studio 2015
+      image_flag: 2017
+    - image: Visual Studio 2017
+      image_flag: 2015
 before_build:
 - nuget restore
 build:
   project: PerfView.sln
   verbosity: minimal
+test_script:
+- .\packages\OpenCover.4.6.519\tools\OpenCover.Console.exe -register:user -target:"%xunit20%\xunit.console.x86.exe" -targetargs:"C:\projects\perfview\src\TraceEvent\Ctf\CtfTracing.Tests\bin\%configuration%\Tests.dll -appveyor" -returntargetcode -excludebyattribute:*.ExcludeFromCodeCoverage* -excludebyfile:*\*Designer.cs -hideskipped:All -output:.\CtfTracing_coverage.xml
```

Comment:
❗️ Need to add a test for the LinuxTracing assembly.
❗️ Need to add a filter because nothing was included in the report.